source/AI.cpp

```diff
@@ -1737,7 +1737,21 @@ void AI::Attack(Ship &ship, Command &command, const Ship &target)
 	
 	// Deploy any fighters you are carrying.
 	if(!ship.IsYours())
-		command |= Command::DEPLOY;
+	{
+		// Conserving ships only deploy fighters if they have lost 50% of shields
+		// or hull, or if they are outgunned.
+		bool beConserving = false;
+		if(ship.GetPersonality().IsConserving())
+		{
+			beConserving = (ship.Hull() + ship.Shields() > 1.5);
+			auto ait = allyStrength.find(ship.GetGovernment());
+			auto eit = enemyStrength.find(ship.GetGovernment());
+			if(ait != allyStrength.end() && eit != enemyStrength.end() && ait->second < eit->second)
+				beConserving = false;
+		}
+			if(!beConserving)
```

Comment:
This and the line below it are indented one level too many.

Reply:
This must be a real struggle.

Reply:
didn't I fix that?